circleci.1m.rb

```diff
@@ -27,6 +27,7 @@
   'canceled' => '#898989',
   'scheduled' =>'#AC7DD3',
   'no_tests' => 'black',
+  'not_running' => 'black',
```

Comment:
I also added 'not_running' to the `$any_running` check, otherwise the icon would be red. Not sure if that is wanted behaviour ;)